### PR TITLE
Enable AllowAllAzureServices by allowing start and end ip being 0.0.0.0

### PR DESCRIPTION
**Pull request by QingqingYuan**
Updated the parameter description only. 
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [na ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ y] Each command and parameter has a meaningful description.
- [ na] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by troydai** [comment]
Please fix the code style to recover from CI failure.

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py [hunk 1]
```diff
@@ -449,6 +449,15 @@
             az webapp create -g MyResourceGroup -p MyPlan -n MyUniqueAppName --runtime "node|6.2" --deployment-local-git
 """
 
+helps['webapp update'] = """
```
Comment (Suna-MS) [inline src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py]: Why do you have those changes? Please remove all changed files that are not related to MySQL/PG AllowAzureService
